config-model/src/main/resources/schema/admin.rnc

```diff
@@ -105,6 +105,7 @@ ClusterControllers = element cluster-controllers {
 }
 
 LogForwarding = element logforwarding {
+    attribute unconditional { xsd:boolean }? &
```

Comment:
"all"? "include-admin"?

Reply:
include-admin sounds good.  it was mostly added in case it was needed for some unit test :-)